python/py-triangle/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        drufat triangle 20160203 v
+
+name                py-triangle
+
+categories-append   science math
+
+platforms           darwin
+supported_archs     noarch
+license             LGPL
```

Comment:
Please specify the version of LGPL, and if it's 'or any later version' append a `+`.